@here/harp-lines/lib/Lines.ts

```diff
@@ -215,8 +215,10 @@ export function createLineGeometry(
         tmpTangent0.normalize();
         geometry.vertices.push(tmpTangent0.x, tmpTangent0.y, tmpTangent0.z);
         tmpVertices.push(tmpTangent0.x, tmpTangent0.y, tmpTangent0.z);
+        tmpNormal.add(center); // tmpNormal contains world position
+        projection.surfaceNormal(tmpNormal, tmpNormal);
```

Comment:
This could be heavily optimized if we know the projection type before hand, wright? For planar projection the normal will be position independent (assuming the we do not consider terrain curvature - which is true for now, yes?).

Reply:
I agree it could be optimized but at what cost? It is an internal property of the projection whether the normal is constant or not. Exposing this knowledge here could cause issues in the future (and was already causing the issue at hand).

Reply:
So can we assume that all Planar projections will have the constant normal? I saw it in many places in the code anyway... Taking into account that we create a lot of lines, this simplification may be worth some if-ology.